models/migrations/v93.go

```diff
@@ -9,7 +9,7 @@ import "github.com/go-xorm/xorm"
 func addEmailNotificationEnabledToUser(x *xorm.Engine) error {
 	// Issue see models/user.go
 	type User struct {
-		EmailNotificationsEnabled bool `xorm:"NOT NULL DEFAULT true"`
+		EmailNotificationsEnabled bool `xorm:"DEFAULT true"`
```

Comment:
Why remove `NOT NULL`?

Reply:
Sorry, that is from back when that was still a bool. I'll put it back on.